src/configs/strings/en.json

```diff
@@ -842,5 +842,9 @@
   "ORGANIZATION_NAME_TITLE": {
     "message": "Team Name",
     "description": "Organization list column header"
+  },
+  "LINK": {
+    "message": "Link",
+    "description": "URL"
```

Comment:
Let's just change this description to be a bit more specific:
`Button text: describes a URL leading to another page`